src/typeahead/typeahead-window.ts

```diff
@@ -56,13 +56,13 @@ export class NgbTypeaheadWindow {
    */
   @Output('select') selectEvent = new EventEmitter();
 
-  getActive() { return this.results[this._activeIdx]; }
+  getActive() { return this.results[this.activeIdx]; }
 
-  markActive(_activeIdx: number) { this._activeIdx = _activeIdx; }
+  markActive(activeIdx: number) { this.activeIdx = activeIdx; }
```

Comment:
If a given component is not exported from our module we don't care about `@internal` too much since we will never have API docs for those components. But still good to have it.

If a given component doesn't have `exportAs` we don't care much eather as for those components don't have methods description as part of their API. So in this sense every method called from a template should be marked as `@internal`. We don't care too much today, but will be good to have things clean if we decide to export a given component one day.

Hope it clarifies.